controllers/velero_test.go

```diff
@@ -133,7 +134,7 @@ func TestVeleroReconciler_buildVeleroDeployment(t *testing.T) {
 							Containers: []corev1.Container{
 								{
 									Name:            common.Velero,
-									Image:           getVeleroImage(),
+									Image:           fmt.Sprintf("%v/%v/%v:%v", os.Getenv("REGISTRY"), os.Getenv("PROJECT"), os.Getenv("VELERO_REPO"), os.Getenv("VELERO_TAG")),
```

Comment:
We want this to use `getVeleroImage()` still. For this test it can pass an empty CR and it should just return the default